src/program.ts

```diff
@@ -4273,9 +4275,8 @@ export class Class extends TypedElement {
     var instance: Class | null = this;
     do {
       let overloads = instance.overloads;
-      if (overloads) {
-        let overload = overloads.get(kind);
-        if (overload) return overload;
+      if (overloads != null && overloads.has(kind)) {
```

Comment:
Another fix within the `Map#has` before `Map#get` wasnt-even-conceived-by-someone category. No error in JS, entirely justified fireworks in Wasm.